docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
My parser uses it to get the type information  (and produces these files: https://github.com/fab1an/kotlin-electron-api/blob/master/build/api_docbook/browser-window.source.normalized.xml), but I can just take "the second word" too